src/diff/index.js

```diff
@@ -258,9 +258,16 @@ function diffElementNodes(dom, newVNode, oldVNode, context, isSvg, excessDomChil
 					}
 				}
 			}
+			if (newVNode.props.dangerouslySetInnerHTML || oldProps.dangerouslySetInnerHTML) {
```

Comment:
I'm not sure if this would be the right path - to extract the `innerHTML` part from props. Have you tried to do the _hydration_ mode, @developit suggested on slack?

Reply:
That would mean that we'd somehow have to get into line 246 when old was set and new is null right?

Reply:
Storing `dangerouslySetInnerHTML` in variables is `5 B` smaller than `vnode.props.dangerouslySetInnerHTML` 🎉 

```js
let newHtml = newVNode.props.dangerouslySetInnerHTML;
let oldHtml = oldProps.dangerouslySetInnerHTML;
if (newHtml || oldHtml) {
  // Avoid re-applying the same '__html' if it did not changed between re-render
  if (!newHtml || !oldHtml || newHtml.__html!=oldHtml.__html) {
    dom.innerHTML = newHtml && newHtml.__html || '';
  }
}
```

Reply:
@marvinhagemeister is this solution acceptable because as stated by @cristianbote, developit also mentioned one about entering hydration, personally I'm not following that one all too well to be honest.

Reply:
@JoviDeCroock The hydration optimisation @developit talked about in slack can and should happen in a separate PR. This one is just for fixing #446 which it does wonderfully 👍